stan/math/prim/fun/ones_vector.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef STAN_MATH_PRIM_FUN_ONES_VECTOR_HPP
+#define STAN_MATH_PRIM_FUN_ONES_VECTOR_HPP
+
+#include <stan/math/prim/err.hpp>
+#include <stan/math/prim/fun/Eigen.hpp>
+
+namespace stan {
+namespace math {
+
+/**
+ * Return a vector of ones
+ *
+ * @param K length of the vector
+ * @return A vector of length K with all elements initialised to 1.
+ */
```

Comment:
We are very inconsistent in the formatting of doxygen comments. Most I've seen don't use a period, but I tend to format the `@return` as a full sentence, so I capitalize and put a period at the end.

Reply:
You're right that the doc on the parameters and return values is inconsistent.  My comment was about the top-level doc string, not the return or param doc!

But while we're on that topic... The typographically correct thing to do for `@param` and `@return` is to use a noun phrase that describes the input or output and thus not use any capitalization or periods.

In the example above, "A vector of length K with all elements initialised to 1" is a noun phrase, not a sentence.  You can tell because there's no main verb.  The only verb is `initialised`, which appears in the subordinate clause "with all elements initialized to 1", which is a modifier of the head noun "vector".